boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15_cpuapp.dts

```diff
@@ -105,6 +105,7 @@
 };
 
 &grtc {
+	child-owned-channels = <7 8 9 10 11>;
```

Comment:
`child-owned-channels = <8 9 10 11>;`

Reply:
Talked internally and taken a decision to leave the 7-11 channels allocated.